wordpress-mobile/WordPressAuthenticator-iOS
# UL&S: SignupMagicLinkViewController: display the UI

Ref. #345 
Testing PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14633

This PR navigates the user to the "magic link sent!" view in the Unified Signup flow. Ensure the UI matches the design. The "Open Mail" button is not yet implemented.

**Design**
<img width="347" alt="Screen Shot 2020-08-12 at 11 30 56 AM" src="https://user-images.githubusercontent.com/1062444/90179831-af491000-dd73-11ea-8ebf-55705c690f79.png">


### To test
1. Check out this branch
2. [Set the email](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/blob/develop/WordPressAuthenticator/Unified%20Auth/View%20Related/Sign%20up/UnifiedSignUpViewController.swift#L26) to a valid, test email address (you can use the plus modifier on your email address, but the gravatar image won't appear if you do that).
3. `bundle exec pod install`
4. Build and run
5. Check out the WPiOS testing PR
6. In the WPiOS testing PR, enable the `.unifiedSignup` feature flag.
7. In the WPiOS testing PR, set the podfile to local pod development.
8. `rake dependencies`
9. Build and run
10. Sign up for WordPress.com > Sign up with Email > Send Link by Email. 

**Expected:** the button navigates you to the "magic link sent" view, which looks like this:

<a href="https://user-images.githubusercontent.com/1062444/90188013-60ee3e00-dd80-11ea-83f2-4dc791578282.png"><img src="https://user-images.githubusercontent.com/1062444/90188013-60ee3e00-dd80-11ea-83f2-4dc791578282.png" width="350" /></a>
